### DISCUSSION TITLE
Installation without admin autorisation

### DISCUSSION THREAD

**Discussion by Guerinpem** [xournalpp/xournalpp#2983] [Q&A] [answered]
I can't install xournal on the laptop from my work (and can't ask to my administrator to install it) because I don't have admin autorisation. 
Is there a way to install it without this autorisation ?

**Comment by rolandlo** [comment]
What operating system do you have at work? If it is Linux you may want to try the AppImage.

**Reply by Guerinpem** [reply]
I'm using win10.

**Comment by rolandlo** [comment]
Hmm, then I wouldn't know of a comfortable way. In principle you could use a permanent live USB stick with some Linux system on it. But that's probably not what you want.

**Comment by Guerinpem** [comment, marked as answer]
I found a way !
I installed xournalpp on my personnal computer by choosing the installation file on a usb key.
Then, I copied the file on my professional laptop and create a shortcut of xournallpp.exe to my desktop.

**Reply by Technius** [reply]
Nice! To make this easier in future versions, we plan on 1) updating the installer to allow non-admin installations; and 2) creating a "portable" version that are just unzip-and-run without requiring installation.
